pkg/controller/machine/controller.go

```diff
@@ -71,6 +71,12 @@ const (
 	unknownInstanceState = "Unknown"
 
 	skipWaitForDeleteTimeoutSeconds = 1
+
+	PausedCondition machinev1.ConditionType = "Paused"
+
+	PausedConditionReason = "AuthoritativeAPINotMachineAPI"
+
+	NotPausedConditionReason = "AuthoritativeAPI is set to MachineAPI"
```

Comment:
Are these exported for a particular reason?

Reply:
Hmm - I'm using the same reasons and condition across the machine and machineset controllers. I could always put them somewhere common, although I'm not sure where's best to do that :)

Reply:
That's ok, maybe just group them together somehow and add a comment explaining why they are public